frontend/src/lib/templates/basicPage/BasicPage.svelte

```diff
@@ -0,0 +1,121 @@
+<script lang="ts">
+  import {_} from 'svelte-i18n';
+  import {Page} from '../page';
+  import type {BasicPageProps} from './BasicPage.type';
+
+  type $$Props = BasicPageProps;
+
+  export let title: $$Props['title'];
+  export let asideClass: $$Props['asideClass'] = 'text-secondary text-center';
+  export let primaryActionsLabel: $$Props['primaryActionsLabel'] = $_('primaryActionsLabel');
+
+  // Merge restProps classes
+  $$restProps.contentClass = `gap-y-lg ${$$restProps.contentClass ?? ''}`;
+</script>
+
+<!--
+@component
+The basic template for pages in both the Voter and Candidate Apps.
+
+The content is provided in named slots with the default slot contents forming
+the main content.
+
+Use the properties to add to the classes of the elements containing the slots,
+to define some ids, pass Aria labels and show an optional progress bar in the
+header. You can also pass any valid properties of the `<Page>` template this
+is based on.
+  
+### Properties
+
+- `title`: The required title of the page.
+- See `BasicPage.type.ts` or code completion info.
+
+### Slots
+
+- default: main content of the page
+- `header`: content for the secondary actions displayed on the right
+  side of the `<header>`
+- `aside`: optional content for the complementary notification displayed at the
+  top of the page, right below the `<header>`
+- `hero`: an optional hero image
+- `heading`: optional content for the `<hgroup>` block, defaults to a `<h1>` 
+  element containing the required `title` property
+- `primaryActions`: optional content for the primary actions displayed at the
+  bottom of the page
+
+### Usage
+
+```tsx
+  <BasicPage title="Let's start!">
+
+    <HeroEmoji slot="hero">🚀</HeroEmoji>
+
+    <svelte:fragment slot="aside">
+      <Icon name="check"/> Your constituency is <strong>Loremipsum</strong>
+    </svelte:fragment>
+
+    <svelte:fragment slot="heading">
+      <p class="text-accent">Preheading</p>
+      <h1>Let's Start!</h1>
+    </svelte:fragment>
+
+    <svelte:fragment slot="headerActions">
+      <Button on:click={showInfo} variant="icon" icon="info" text="Show info"/>
+    </svelte:fragment>
+
+    <p class="text-center">
+      This is the body text of the page.
+    </p>
+    
+    <svelte:fragment slot="primaryActions">
+      <Button href="/next" variant="main" icon="next" text="Continue"/>
+    </svelte:fragment>
+    
+  </BasicPage>
+```
+-->
+
+<Page {title} {...$$restProps}>
+  <!-- Header -->
+  <slot name="header" slot="header" />
+
+  <!-- Default slot for Page starts -->
+
+  <!-- Aside -->
+  {#if $$slots.aside}
+    <aside class={asideClass}>
```

Comment:
True, now that I reread the `aside` specification, it's not the best one for this. The intended use for it, though, is not as a complementary title, but an additional note or tip relating to the main content. I'll change it to have a role of [note](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/note_role), which can be overriden.

I'll also wrap everything in `main`, which was the way I originally had it :D but because of this being an `aside`, I couldn't have it inside `main`…